EWSType/FindMailboxStatisticsByKeywordsResponseMessageType.php

```diff
@@ -32,43 +19,4 @@ class EWSType_FindMailboxStatisticsByKeywordsResponseMessageType extends EWSType
      * @var EWSType_MailboxStatisticsSearchResultType
      */
     public $MailboxStatisticsSearchResult;
-
-    /**
-     * Provides a text description of the status of the response.
-     *
-     * @since Exchange 2013
```

Comment:
I mean the `@since` tag doesn't really enforce any restriction based on the version of exchange the WSDL or SOAP client you're using. Either way the object is going to be exactly the same structurally.

You'd almost have to make a separate class each time a new property is introduced right? Or use some ridiculousness with annotations.